docs/en/ingest-management/release-notes/release-notes-8.2.asciidoc

```diff
@@ -0,0 +1,248 @@
+// Use these for links to issue and pulls. 
+:kib-issue: https://github.com/elastic/kibana/issues/
+:kib-pull: https://github.com/elastic/kibana/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.2.0>>
+
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.2.0 relnotes
+
+[[release-notes-8.2.0]]
+== {fleet} and {agent} 8.2.0
+
+Review important information about the {fleet} and {agent} 8.2.0 release.
+
+[discrete]
+[[breaking-changes-8.2.0]]
+=== Breaking changes
+
+There are no breaking changes for {fleet} or {agent} in this release.
+
+[discrete]
+[[known-issues-8.2.0]]
+=== Known issues
+
+[[known-issue-issue-kibana-130758]]
+.During package upgrade, legacy component templates are not removed
+[%collapsible]
+====
+
+*Details* 
+
+In 8.2.0, when you upgrade an existing package, legacy `@settings` and
+`@mappings` component templates are not removed. These templates are deprecated
+in favor of the new `@package` component template. 
+
+*Impact* +
+
+The legacy templates do not cause issues, but you might want to deleted them
+because they are no longer used.
+
+To find and delete legacy component templates:
+
+. In {kib}, go to *Stack Management > Index Management* and click
+*Component Templates*.
+
+. Click `Not in use` to filter out component templates that are still in use.
+
+. If the list contains component templates that end in `@mappings` or
+`@settings`, use the Actions menu to delete them.
+
+====
+
+//REVIEWERS: Are there any //deprecations we need to document here? It sounds
+//like some component templates were deprecated, but not sure when.
```

Comment:
The component templates would not really be a deprecation. I can't think of anything from Fleet side.